data-access-layer/cf/TokenIssuer.js

```diff
@@ -63,6 +63,38 @@ class TokenIssuer {
       .catch(() => this.login())
       .then(result => this.updateTokenInfo(result).accessToken);
   }
+
+  scheduleNextRequestAccessToken(clientId, clientSecret) {
+    const delay = this.tokenInfo.accessTokenExpiresIn - 15;
```

Comment:
store 15 in constants and also indicate what unit of measure it is . - mins , secs etc. .

Reply:
+1